about.html

```diff
@@ -433,13 +455,13 @@ <h4 class="title">
                             Information
                         </div>
                         <small class="text mb-10 d-block">
-                            x Thatcher Road St, Brookly, <br> x,sydney,nsw,Australia
+                            348 Waterloo Road, GREENACRE, NSW, 2190 <br> 
                         </small>
                         <small class="text mb-10 d-block">
-                            +0011111222
+                            (+61)0417 90 6971
                         </small>
                         <small class="text d-block">
-                            contact@xxx.co
+                            pleno@bigpond.com
```

Comment:
Looks like we need to make this clickable. Can you please wrap this email address inside an `<a>` tag with a `href="mailto:<email-address>"` property?